docker-compose.yml

```diff
@@ -86,3 +86,16 @@ services:
             - "/etc/passwd:/etc/passwd:ro"
         depends_on:
             - irods-catalog-provider
+
+    minio:
+        image: quay.io/minio/minio:RELEASE.2022-10-29T06-21-33Z
```

Comment:
Is this a reliable source for this image?
Is there a reason for not using docker hub?

Reply:
Yeah that's Minio's official source, copied that link verbatim, straight from their latest official docs.

Quay is operated by Red Hat, no pull limits. Haven't checked if Minio also publishes on Docker Hub - feel free to change to Docker Hub if more convenient (for folks who have to apply for whitelisting of outgoing destinations).

Reply:
https://hub.docker.com/r/minio/minio appears live, updated 4h ago, with 1B+ downloads.

Reply:
Image location chagned to the same image version on Docker Hub.